src/screens/AnonymousChatScreen/index.tsx

```diff
@@ -88,6 +88,8 @@ const AnonymousChatScreen = () => {
     return <BaseChatItem type={ANONYMOUS} item={item} index={index} />;
   }, []);
 
+  const betterSocialMember = selectedChannel?.rawJson?.better_channel_member;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new variable `betterSocialMember` has been introduced. Ensure that this variable is used somewhere in the code, otherwise, it's just adding unnecessary complexity.
```diff
+  const betterSocialMember = selectedChannel?.rawJson?.better_channel_member;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->